src/components/shared/forms/hyperlink.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { joinClasses } from 'utilities';
+
+import './styles/hyperlink.css';
+
+export const Hyperlink = (props) => {
+  const { children, className, href, target } = props;
+
+  if (href == null) return null;
+  return (
+    <a className={joinClasses('hyperlink', className)} href={href} target={target}>
```

Comment:
I don't think you need to pull out href and target specifically.  They should get  passed on with {...props} right?
```html
<a className={joinClasses('hyperlink', className)} {...props}>
```